rpa-builder/6.1/modules/ROOT/pages/toolbox-mail-operations-mail-session-outlook-with-oauth.adoc

```diff
@@ -0,0 +1,189 @@
+** *Client ID*
++
+To authenticate with the OAuth Host, an Azure application has to be defined in the customer's space. This Azure application has an id, that is unique in the entire OAuth Host space and allows a user to explicitly grant or revoke access for this Azure application to his account. For more information on how to create this Azure application for Outlook, see here: https:// learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app
+** *Client Secret*
++
+This secret allows RPA Builder to prove to the OAuth Host, that it has received permission to access user accounts on behalf of the registered Azure application. Without this permission, authentication is not possible.
+** *Redirect Uri*
++
+This is the URI to which the OAuth Host redirects after the user has completed the authentication attempt. This redirect URI has to be registered with the Azure application. Only if the entered redirect URI matches one of the registered URI, the OAuth Host will allow the authentication.
+** *Scopes*
++
+The scopes define which permissions the user grants to the Azure application. By default, RPA Builder requests the following permissions:
+
+*** *offline_access*
++
+(*Required*) Enables access via a refresh token, which you can use to repeatedly log in to the email services without requiring user interaction each time.
+*** *https://outlook.office.com/IMAP.AccessAsUser.All*
++
+(*Required* when using IMAP) Enables reading and moving emails from the Outlook IMAP server.
+*** *https://outlook.office.com/POP.AccessAsUser.All*
++
+(*Required* when using POP3) Enables reading emails from the Outlook POP3 server.
+*** *https://outlook.office.com/SMTP.Send*
++
+(*Required* when using SMTP) Enables sending emails from the Outlook SMTP server.
+*** *openid email*
++
+Enables RPA Builder to automatically detect the email account used to log in to the Azure application. If this scope is omitted, you must provide an email in the wizard.
++
+** *Email address*
++
+The email address fills automatically if RPA Builder can detect it from the authentication attempt. If the *openid email* scope is omitted, you must provide an email address manually.
+
+* *Incoming Mail Operation* Settings
+** *Protocol used*
++
+(Mail Protocol) Displays the communication protocol to use: IMAP, Exchange, or POP3.
+**  *Encryption used*
++
+(Encryption) Displays the encryption protocol to use: None, SSL/TLS, or STARTTLS.
+** *Server address*
++
+(String) The server address entered in the wizard.
+** *Server Port*
++
+(Integer, Default: `143`) The selected port.
+
+* *Outgoing Mail Operation* Setting
+**  *Encryption used*
++
+(Encryption) Displays the encryption protocol to use: None, SSL/TLS, or STARTTLS.
+** *SMTP Server address*
++
+(String) The URL of the selected SMTP email server.
+** *SMTP Server Port*
++
+(Integer) Displays the selected port.
+** *Send test mail to*
++
+(String) Email address to which the test email is to
```

Comment:
Minor nit: some of the other descriptions start with "The." For consistency, change this to "The ..." or remove "The" from the start of all descriptions.